### PR TITLE
fix #8013 feat(nimbus): Nimbus logo

### LINKED ISSUE #8013: lets add logo \o/
┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-2969)

### PR DESCRIPTION
**Pull request by yashika-khurana**
Because

* We have an awesome logo now 
Asset 1

This commit

* Doesn't wait to get this new logo added \o/
Screenshot 2022-12-08 at 11 08 06 AM

fix #8013

### CODE REVIEW

**Comment by mikewilli** [comment]
Aw yea, new logo! While you're adding it, should we try it out as a favicon as well?

File: app/experimenter/nimbus-ui/src/components/PageHome/index.tsx [hunk 1]
```diff
@@ -13,6 +13,7 @@ import {
   useSearchParamsState,
 } from "../../hooks";
 import { ReactComponent as DownloadIcon } from "../../images/download.svg";
+import { ReactComponent as CreateNewLogo } from "../../images/logo.svg";
```
Comment (eliserichards) [inline app/experimenter/nimbus-ui/src/components/PageHome/index.tsx:16]: I think we can just call this Logo. WDYT?

```suggestion
import { ReactComponent as Logo } from "../../images/logo.svg";
```

**Review by eliserichards — DISMISSED** [review]
I'm so excited to get this merged!! I do think we should use the version of the image without the "nimbus" text on it though, since we already have the page title right next to it?

**Comment by yashika-khurana** [comment]
> Aw yea, new logo! While you're adding it, should we try it out as a favicon as well?

@mikewilli that's an amazing idea, I completely agree, let me file a ticket for that I really want this one to land earliest :D

**Comment by yashika-khurana** [comment]
@mikewilli @eliserichards 
❤️ 
Screenshot 2022-12-08 at 12 04 25 PM

**Review by eliserichards — APPROVED** [review]
image

File: app/experimenter/nimbus-ui/src/components/PageHome/index.tsx [hunk 2]
```diff
@@ -106,8 +106,17 @@ const PageHome: React.FunctionComponent<PageHomeProps> = () => {
...
 
-      <h2 className="mb-3 text-left">Nimbus Experiments </h2>
-      <Alert variant="primary" className="mb-4">
+      <Logo
+        width="100"
+        height="85"
+        fill="white"
+        dominantBaseline="start"
+        role="img"
+        aria-label="download icon"
```
Comment (mikewilli) [inline app/experimenter/nimbus-ui/src/components/PageHome/index.tsx:115]:
```suggestion
        aria-label="nimbus logo"
```

**Review by mikewilli — CHANGES_REQUESTED** [review]
Just one small thing, can you update the aria-label? Otherwise looks great!

**Review by mikewilli — APPROVED** [review]

**Pull request merged**